pkg/data/values_test.go

```diff
@@ -0,0 +1,108 @@
+package data
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetValue(t *testing.T) {
```

Comment:
Please run the test and all its subtests in parallel.

Reply:
done